### ISSUE TITLE
Add ability to scale up number of Heroku workers

### ISSUE THREAD

**Issue by matthewcornell** [reichlab/forecast-repository#177]
Currently (see [resources](https://dashboard.heroku.com/apps/reichlab-forecast-repository/resources)) we have: one `web` dyno and one `worker` dyno, as configured in our [Procfile](https://github.com/reichlab/forecast-repository/blob/master/Procfile). 

```
web: gunicorn forecast_repo.wsgi --log-file=-
worker: python3 manage.py rqworker default
```

However, recall that we are using `Hobby` dynos which allow:

     For each app, developers can configure up to 10 process types, each running on one (1) Hobby dyno.

This is different from the horizontal scaling that much more expensive Professional dynos provide. After investigation, I think our best scaling option using Hobby dynos is to change the Procfile to have more worker process types, say 7 of them (leaving 3 for web plus a few scheduler workers).

**Comment by matthewcornell** [comment]
Re-opening this b/c I saw this kind of error when testing scaling up to 7 workers:

    redis.exceptions.ConnectionError: max number of clients reached

After a little exploring on staging, I learned:
- adding workers increases # clients, roughly: 1,6  2,9  3,10  4,13  5,16  6,13  7,18
- client counts vary
- scaling up slowly (adding one worker at a time) *might* be better than 1->7 directly
- I believe that there is some connection pooling happening, as #clients roughly tracks #dynos
  = I'm not very confident, given it's complicated and I don't know much yet

There are non-Heroku redis plans that have more memory and clients, e.g.,
- https://elements.heroku.com/addons/rediscloud
  = 30MB:  Free   30
  = 100MB: $10/mo 256
- vs heroku: https://elements.heroku.com/addons/heroku-redis
  = Hobby Dev: Free    25MB 20  # our current one
  = Premium 0: $25/mo  50MB 40

**Comment by matthewcornell** [comment]
Switched production from https://elements.heroku.com/addons/heroku-redis to https://elements.heroku.com/addons/rediscloud . Left staging on heroku-redis b/c rediscloud only allows one free instance per account. Closed by 047f9f3dc1cfbacaabaf87e0007a9347f2e311ad
